docs/release-notes/release-v0.8.md

```diff
@@ -0,0 +1,302 @@
+
+Added the `utils parse` command to parse a Kuneiform file and output the schema as JSON.
+
+Added the `utils decode-tx` command to decode a serialized transaction provided as a base64 string.
+
+Many of the fields in the JSON objects returned by `query-tx` and other commands with the `--output json` option are renamed with consistent snake case formatting. See the See [UPGRADING.md](https://github.com/kwilteam/kwil-db/blob/release-v0.8/UPGRADING.md) for details on changes to the `core` module types and their JSON tags.
+
+The `account balance` command may now be given a `--pending` flag to return account information that includes changes that consider any unconfirmed transactions. This is useful for determining the next nonce for an account when broadcasting multiple transaction per block.
+
+#### Node Application (`kwild`) Changes
+
+The default setting for the admin service (`admin_listen_addr`) is now `"/tmp/kwild.socket"`.
+
+The `jsonrpc_listen_addr` setting replaces the `grpc_listen_addr` setting. The default value for `jsonrpc_listen_addr` is `0.0.0.0:8484` (listen on port 8484 on all network interfaces).
+
+In addition to standard output, `kwild` also writes the log to `"kwild.log"` in the application's root directory by default. The `output_paths` setting is used to change this.
+
+Two new timeouts are added:
+
+- `rpc_timeout` sets a timeout on requests on the user RPC servers
+- `db_read_timeout` sets a timeout on database reads initiated by the user RPC service
+
+To support [network migrations](#network-migrations), the `genesis_state` setting is added to specify the path to the snapshot file to restore the database from.
+
+A new `app.snapshots` section is added to configure snapshot creation, which supports other nodes using "state sync" when joining a network.
+
+A new `chain.statesync` section is added to configure the use of "state sync" if the node is joining a network.
+
+A new `broadcast_tx_timeout` setting is added to control the timeout used when a transaction is broadcast with the `--sync` flag to wait for it to be mined. If the timeout is exceeded, the RPC user should query the transaction status until it is confirmed. The default timeout is 15 seconds.
+
+See the [Node documentation](https://docs.kwil.com/docs/daemon/config/settings) for more information on the added and changed settings.
+
+### SDK (`core` module)
+
+This release of Kwil DB uses version 0.2.1 of the `core` Go module.
+
+A Go application that supports Kwil DB v0.8.0 should be developed using the `core` module specified in the go.mod as follows:
+
+```go
+require github.com/kwilteam/kwil-db/core v0.2.1
+require github.com/kwilteam/kwil-db/parse v0.2.1 // to parse Kuneiform files
```

Comment:
Mmmm,  I should have done a section like this for 0.9.  I may circle back on that.